aries-backchannels/acapy/requirements-bbs.txt

```diff
@@ -0,0 +1 @@
+aries-cloudagent[indy, bbs]@git+https://github.com/animo/aries-cloudagent-python@cwu-bbs-main
```

Comment:
I think we should just add this to acapy-main (requirements-main.txt, Dockerfile.acapy-main) rather than adding a new "bbs" version of the build

Reply:
(for me acapy-main was breaking without the ursa bbs library installed)

Reply:
That's the idea! Will update the PR